### ISSUE TITLE
Map is not shown if zero coords are set

### ISSUE THREAD

**Issue by ildar170975** [nathan-gs/ha-map-card#9]
Consider this code:
```
type: vertical-stack
cards:
  - type: custom:map-card
    x: 10
    y: 10
  - type: custom:map-card
    x: 0
    y: 0
```
The 2nd card with zero coords not shown:

![изображение](https://github.com/nathan-gs/ha-map-card/assets/71872483/2d0ca86a-7d1d-4190-83d8-78921dd75d65)

Have to add some digits after a point:
```
  - type: custom:map-card
    x: 0.001
    y: 0.001
```
![изображение](https://github.com/nathan-gs/ha-map-card/assets/71872483/a240c7f4-edf8-4bb4-8dc1-197c5d9815b3)
(map is zoomed to demonstrate a correct location)

0.4.0
HA 2024.3.1
Chrome 123.0.6312.59 (Win10x64)

**Comment by nathan-gs** [comment]
Fixed in v0.4.2

**Comment by ildar170975** [comment]
Confirmed, the problem gone.
Thank you!
